### ISSUE TITLE
[BUG/ISSUE] Compilation failed on patch/13.2.1 with GNU 8

### ISSUE THREAD

**Issue by LiamBindle** [geoschem/geos-chem#878]
@yantosca On the patch/13.2.1 with GNU 8 (re: https://github.com/geoschem/geos-chem/commit/9580101a582418c502526064e83a5f9055a46ac1) I'm getting 

```
[ 94%] Building Fortran object src/GCHP_GridComp/GEOSChem_GridComp/geos-chem/GeosCore/CMakeFiles/GeosCore.dir/input_mod.F90.o
/scratch1/fs1/rvmartin/bs6/liam.bindle/NativeMet/GCHP/src/GCHP_GridComp/GEOSChem_GridComp/geos-chem/GeosCore/mercury_mod.F90:2767:21:

              EHg0_geo(I,J,:)   = 0.0_fp
                     1
Error: Rank mismatch in array reference at (1) (3/2)
make[2]: *** [src/GCHP_GridComp/GEOSChem_GridComp/geos-chem/GeosCore/CMakeFiles/GeosCore.dir/mercury_mod.F90.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [src/GCHP_GridComp/GEOSChem_GridComp/geos-chem/GeosCore/CMakeFiles/GeosCore.dir/all] Error 2
make: *** [all] Error 2
```

Do you know a fix?

**Comment by LiamBindle** [comment]
No problem! Thanks!

**Comment by yantosca** [comment]
This is now fixed in ae7afb8682eeb0f0453d442b44935adfe3b2359d.  Also referencing Issue #847.

**Comment by LiamBindle** [comment]
Thanks Bob!
